<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="content-type" content="text/html;charset=iso-8859-1"/>
<link type="text/css" rel="stylesheet" href="/benchmarkservice.css"/>
<script src="/scripts.js" type="text/javascript"></script>
<title>Benchmark Selection <DVAR name="tit"></title>
</head>
<body>
<div id="main">
<h2>Benchmark Selection for <DVAR name="tit"></h2>
Select from the available benchmarks and against which standard 
orthlogy inference project you would like to compare your predictions
<form action="gateway.pl" method="post">

<fieldset><legend>Phylogeny-Based Definition Tests</legend>
  <div class="notes"><h4>Phylogeny-Based Definition</h4>
   <p>According to the phylogenetic definition, two homologous genes are orthologs if they diverged through a speciation event. Therefore, the phylogenetic tree of a set of orthologs (a set of genes in which any pair is orthologous) has by definition the same topology as the corresponding species tree.</p>
  </div>
  <div>
   <label for="pTTEuk">Species Tree Discordance</label>
    <fieldset>
     <input type="checkbox" name="pTTEuk1" class="inputCheckbox" <DVAR name="hasEukaryota">/>Eukaryotic <br/>
     <input type="checkbox" name="pTTFun1" class="inputCheckbox" <DVAR name="hasFungi">/>Fungi <br/>
     <input type="checkbox" name="pTTBac1" class="inputCheckbox" <DVAR name="hasBacteria">/>Bacteria <br/>
    </fieldset>
   <label for="pTTPrj1">Projects to compare with</label>
   <DVAR name="TTprojects">
  </div>
 </fieldset>
 
 <fieldset><legend>Function-Based Definition Test</legend>
  <div class="notes"><h4>Function-Based Definition</h4>
   <p>One of the main application of orthology is the propagation of functional annotation, because orthologs tend to have a similar function.</p>
  </div>
  <div>
   <input type="checkbox" name="pGO1" class="inputCheckbox" checked />Gene Ontology Test<br/>
   <label for="pGOPrj1">Projects to compare with</label>
   <DVAR name="GOprojects">
   
   <fieldset><legend>Restricted to Evidence codes</legend>
    <p>
    <input type="checkbox" name="pGOFilEXP" checked><span name="hint" title="Inferred from Experiment">EXP</span></input>
    <input type="checkbox" name="pGOFilIDA" checked><span name="hint" title="Inferred from Direct Assay">IDA</span></input>
    <input type="checkbox" name="pGOFilIPI" checked><span name="hint" title="Inferred from Physical Interaction">IPI</span></input>
    <input type="checkbox" name="pGOFilIMP" checked><span name="hint" title="Inferred from Mutant Phenotype">IMP</span></input>
    <input type="checkbox" name="pGOFilIGI" checked><span name="hint" title="Inferred from Genetic Interaction">IGI</span></input>
    <input type="checkbox" name="pGOFilIEP" checked><span name="hint" title="Inferred from Expression Pattern">IEP</span></input>
    </p>
    <p>
    <input type="checkbox" name="pGOFilISS"><span name="hint" title="Inferred from Sequence or Structural Similarity">ISS</span></input>
    <input type="checkbox" name="pGOFilISO"><span name="hint" title="Inferred from Sequence Orthology">ISO</span></input>
    <input type="checkbox" name="pGOFilISA"><span name="hint" title="Inferred from Sequence Alignment">ISA</span></input>
    <input type="checkbox" name="pGOFilISM"><span name="hint" title="Inferred from Sequence Model">ISM</span></input>
    <input type="checkbox" name="pGOFilIGC"><span name="hint" title="Inferred from Genomic Context">IGC</span></input>
    <input type="checkbox" name="pGOFilIC" ><span name="hint" title="Inferred by Curator">IC </span></input>
    <input type="checkbox" name="pGOFilRCA"><span name="hint" title="Inferred from Reviewed Computational Analysis">RCA</span></input>
    <input type="checkbox" name="pGOFilTAS"><span name="hint" title="Traceable Author Statement">TAS</span></input>
    <input type="checkbox" name="pGOFilNAS"><span name="hint" title="Non-traceable Author Statement">NAS</span></input>
    </p>
    <p>
    <input type="checkbox" name="pGOFilIEA"><span name="hint" title="Inferred from electronic annotation">IEA</span></input>
    <input type="checkbox" name="pGOFilND" ><span name="hint" title="No biological Data available">ND</span></input>
    </p>
   </fieldset>
   
   <fieldset><legend>Similarity Measure</legend>
    <input type="radio" name="pGOMeasure" value="avg Sim">average Lin Similarity</input><br/>
    <input type="radio" name="pGOMeasure" value="max Sim">maximum Lin Similarity</input><br/>
    <input type="radio" name="pGOMeasure" value="avg Info">average information content</input><br/>
    <input type="radio" name="pGOMeasure" value="max Info">maximum information content</input><br/>
    <input type="radio" name="pGOMeasure" value="avg Schlicker" checked>average Schlicker Similarity</input>
   </fieldset>
  </div>
  <div>
   <input type="checkbox" name="pEC1" class="inputCheckbox" checked />Enzyme Classification (EC) Test<br/>
   <label for="pECPrj1">Projects to compare with</label>
   <DVAR name="ECprojects">
   
   <fieldset><legend>Similarity Measure</legend>
    <input type="radio" name="pECMeasure" value="avg Sim">average Lin Similarity</input><br/>
    <input type="radio" name="pECMeasure" value="max Sim">maximum Lin Similarity</input><br/>
    <input type="radio" name="pECMeasure" value="avg Info">average information content</input><br/>
    <input type="radio" name="pECMeasure" value="max Info">maximum information content</input><br/>
    <input type="radio" name="pECMeasure" value="avg Schlicker" checked>average Schlicker Similarity</input>
   </fieldset>
  </div>
 </fieldset>

 <fieldset>
  <div>
   <input type="hidden" name="f" value="RunTests" />
   <input type="hidden" name="pPID" value="<DVAR name="PID">" />
   <input type="submit" value="Compute &raquo;" />
  </div>
 </fieldset>
</form>
<DVAR name="content">
</div>
</body>
</html>
